Userland/Libraries/LibJS/Tests/builtins/Set/Set.prototype.values.js

```diff
@@ -12,3 +12,36 @@ test("basic functionality", () => {
     expect(it.next()).toEqual({ value: undefined, done: true });
     expect(it.next()).toEqual({ value: undefined, done: true });
 });
+
+describe("keys is an alias for values", () => {
+    test("length", () => {
+        expect(Set.prototype.keys.length).toBe(0);
+    });
+
+    test("basic functionality", () => {
+        const a = new Set([1, 2, 3]);
+        const it = a.keys();
+        expect(it.next()).toEqual({ value: 1, done: false });
+        expect(it.next()).toEqual({ value: 2, done: false });
+        expect(it.next()).toEqual({ value: 3, done: false });
+        expect(it.next()).toEqual({ value: undefined, done: true });
+        expect(it.next()).toEqual({ value: undefined, done: true });
+        expect(it.next()).toEqual({ value: undefined, done: true });
+    });
+});
+
+test("third appears in entries value when 8 are added in set", () => {
```

Comment:
...what?

Reply:
Oh right this tests isn't needed (it the case where I found the `find_smallest_not_below_iterator` problem